### PR TITLE
ICMSLST-2869 - FA-SIL UI Improvements

### PR DESCRIPTION
**Pull request by marcuspp**
Screenshot 2024-10-11 at 12 17 19
Screenshot 2024-10-11 at 12 18 03

### CODE REVIEW

File: web/templates/partial/know-bought-from.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     You will be asked to provide this information after the import licence has been issued.
     An export authority cannot issue a firearms export licence until this information has been provided
   </div>
-{% elif process.know_bought_from == True %}
+{% elif process.know_bought_from == True and not contacts %}
```
Comment (marcuspp) [inline web/templates/partial/know-bought-from.html:6]: Not related but whilst testing I don't think this should be shown if a contact has already been added

File: web/templates/web/domains/case/import/fa-sil/view.html [hunk 2]
```diff
@@ -17,14 +17,11 @@
     <li>
       <a href="#view-certificates">Certificates</a>
     </li>
-    <li>
-      <a href="#view-licence-for">Goods - Licence For</a>
```
[the remark this answers is not in the document]
Comment (MattHolmes123) [reply web/templates/web/domains/case/import/fa-sil/view.html:21]: Why? Whats the harm in having links that take you to each section?

[the remark this answers is not in the document]
Comment (MattHolmes123) [reply web/templates/web/domains/case/import/fa-sil/view.html:21]: Oh I see, it's right next to the tiny goods section
```jinja
{% call application_section("Goods", "view-goods") %}
    {{ application_field(process.commodity_code, "Commodity Code") }}

    <hr />
```

[the remark this answers is not in the document]
Comment (marcuspp) [reply web/templates/web/domains/case/import/fa-sil/view.html:21]: The screen was cluttered and the link offered no value as everything is very close together

File: web/templates/web/domains/case/import/fa-sil/view.html [hunk 3]
```diff
@@ -116,28 +111,16 @@ <h5 id="view-licence-for"> Licence for</h5>
...
-
-    <h5>Section 5 Authorities</h5>
-      {% with read_only = True %}
-        {% include "web/domains/case/import/partials/fa-sil/verified-section5-authorities.html" %}
-      {% endwith %}
-      {% with read_only = True %}
-        {% include "web/domains/case/import/partials/fa-sil/user-section5-authorities.html" %}
-      {% endwith %}
+    <hr /><hr />
   {% endcall %}
```
Comment (MattHolmes123) [inline web/templates/web/domains/case/import/fa-sil/view.html:122]: Not sure about this change. I will test it after lunch.

**Review by MattHolmes123 — CHANGES_REQUESTED** [review]
I need to test something

**Comment by MattHolmes123** [comment]
Added a better screenshot @marcuspp including all 4 types of certificates.

Before:
image

After:
image

**Review by MattHolmes123 — APPROVED** [review]

**Pull request merged**